crates/rattler_networking/src/authentication_middleware.rs

```diff
@@ -4,6 +4,8 @@ use crate::{Authentication, AuthenticationStorage};
 use async_trait::async_trait;
 use base64::prelude::BASE64_STANDARD;
 use base64::Engine;
+use google_cloud_auth::project::create_token_source;
```

Comment:
do we need to make these `use` statements so that they are only enabled when the `google-cloud-auth` feature is enabled?